elide-contrib/elide-dynamic-config-helpers/src/main/java/com/yahoo/elide/contrib/dynamicconfighelpers/parser/handlebars/HandlebarsHelper.java

```diff
@@ -0,0 +1,246 @@
+
+        String type = capitalizeFirstLetter(join.getTo());
+        String fieldName = join.getName();
+        return "public void set" + capitalizeFirstLetter(fieldName) + "(" + type + SPACE + fieldName + ") {" + NEWLINE
+                        + EIGHT_SPACES + "this." + fieldName + " = " + fieldName + ";" + NEWLINE + FOUR_SPACES + "}";
+    }
+
+    /**
+     * Generate getter method.
+     * @param type Type of the field
+     * @param fieldName field Name
+     * @return getter method
+     */
+    public String generateGetterMethod(String type, String fieldName) {
+
+        return "public " + type + " get" + capitalizeFirstLetter(fieldName) + "() {" + NEWLINE + EIGHT_SPACES
+                        + "return " + fieldName + ";" + NEWLINE + FOUR_SPACES + "}";
+    }
+
+    /**
+     * Generate getter method for Dimension type.
+     * @param dim Dimension Object
+     * @return getter method
+     */
+    public String generateDimGetterMethod(Dimension dim) {
+
+        String type = getDimensionType(dim);
+        String fieldName = dim.getName();
+        return "public " + type + " get" + capitalizeFirstLetter(fieldName) + "() {" + NEWLINE + EIGHT_SPACES
+                        + "return " + fieldName + ";" + NEWLINE + FOUR_SPACES + "}";
+    }
+
+    /**
+     * Generate getter method for Join type.
+     * @param join Join Object
+     * @return getter method
+     */
+    public String generateJoinGetterMethod(Join join) {
+
+        String type = capitalizeFirstLetter(join.getTo());
+        String fieldName = join.getName();
+        return "public " + type + " get" + capitalizeFirstLetter(fieldName) + "() {" + NEWLINE + EIGHT_SPACES
+                        + "return " + fieldName + ";" + NEWLINE + FOUR_SPACES + "}";
+    }
+
+    // Security.java helper methods
+    /**
+     * Get the Class Name for Security.
+     * @param name Name of the security rule
+     * @return Capitalize First Letter of Each Word and remove spaces
+     */
+    public String getClassSuffix(String name) {
+
+        String[] nameSplit = name.trim().split(WHITESPACE_REGEX);
+        return String.join(EMPTY_STRING,
+                Arrays.asList(nameSplit).stream().map(str -> capitalizeFirstLetter(str)).collect(Collectors.toList()));
+    }
+
+    /**
+     * Get Variable Name for Security rule.
+     * @param name Name of the security rule
+     * @return Change each word to upper case and replace spaces with underscore
+     */
+    public String getVarName(String name) {
+
+        String[] nameSplit = name.trim().split(WHITESPACE_REGEX);
+        return String.join(UNDERSCORE,
+                Arrays.asList(nameSplit).stream().map(str -> toUpperCase(str)).collect(Collectors.toList()));
+    }
+
+    /**
+     * Get Name of the security role from security rule.
+     * @param name Name of the security rule
+     * @return Last word in lower case
+     */
+    public String getRoleName(String name) {
```

Comment:
The role name should not be extracted from the Rules - but rather the Roles.  As such, we don't need to parse text like "Principal is ADMIN".